UNRELEASED.md

```diff
@@ -10,14 +10,13 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 - `Page > Header` now must pass `content` to the `actions` within `secondaryActions` and `actionGroups` ([#1653](https://github.com/Shopify/polaris-react/pull/1653))
 
-### Deprecations
```

Comment:
Could we add this back?

Reply:
its still there! what i removed was a duplicate :)